vhdbuilder/packer/components.json

```diff
@@ -81,7 +81,8 @@
       "downloadURL": "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "3.1.8"
+        "3.1.8",
```

Comment:
Lets remove 3.1.8 and keep only latest version so that we can avoid getting incidents related to vulnerabilities on old images.